source/vir/src/sst_to_air.rs

```diff
@@ -1072,18 +1072,18 @@ pub(crate) fn exp_to_expr(ctx: &Ctx, exp: &Exp, expr_ctxt: &ExprCtxt) -> Result<
                 ident_apply(&crate::def::to_dyn(trait_path), &args)
```

Comment:
There was a conflicting change here that caused a stack overflow and test failure, so in ToDyn, `exp` can be changed to `e` and `exp_typ` can be changed to `&exp.typ` and `exp_typ` can be removed.